generic_aggregation_tests/tests.py

```diff
@@ -167,7 +205,7 @@ def test_filter_cast(self):
             self.assertEqual(obj.content_object.name, 'apple')
 
 class FallbackTestCase(SimpleTest):
-    def generic_aggregate(self, *args, **kwargs):
+    def generic_annotate(self, *args, **kwargs):
```

Comment:
This is why the fallback version appeared to be equivalent to the normal version in tests -- it wasn't being tested.